### ISSUE TITLE
Coloured letters using /ps setname

### ISSUE THREAD

**Issue by MackySan** [phaed420/PreciousStones#790]
Hi, I am unsure how to get coloured letters into the names of fields. I cant use "§" because it doesnt come up in the chat bar.

The only way I have been able to do it is open preciousstones.db and adding the § symbol myself.
I have read through the documentation and cant find how to do it. (Unless I've missed it).

**Comment by phaed420** [comment]
Use ampersands

**Comment by MackySan** [comment]
Yes, they just appear like this:

![screenshot 2015-03-08 12 49 48](https://cloud.githubusercontent.com/assets/1496742/6544409/5afcb8ce-c5a2-11e4-98f6-9f070c89ca7b.png)

Im using PreciousStones 9.7.1
I am running a Spigot server, I have tried this on versions 1.8 to 1.8.3, no difference.
I also tested on a local server and used PreciousStones as my only plugin. Same problem.
Also no errors come up in the console.
